libr/core/disasm.c

```diff
@@ -3946,6 +3947,25 @@ static void ds_print_ptr(RDisasmState *ds, int len, int idx) {
 #endif
 }
 
+static void ds_print_demangled(RDisasmState *ds) {
+	if (!ds->show_comments || !ds->asm_demangle) {
+		return;
+	}
+	RCore *core = ds->core;
+	RFlagItem *f = NULL;
```

Comment:
no need to initialize, you can assign it below too if you add some braces

Reply:
ok f1680776356163b9c30486497b89525a92a744c1